clang/lib/AST/Interp/IntegralAP.h

```diff
@@ -167,17 +169,13 @@ template <bool Signed> class IntegralAP final {
   }
 
   static bool increment(IntegralAP A, IntegralAP *R) {
-    // FIXME: Implement.
-    assert(false);
-    *R = IntegralAP(A.V - 1);
-    return false;
+    IntegralAP<Signed> One(1, A.bitWidth());
```

Comment:
```suggestion
    static constexpr IntegralAP<Signed> One(1, A.bitWidth());
```
Can we do this so we don't have to create the same `IntegralAP` object every time? (Same suggestion below.)

Reply:
That doesn't work because `APInt` is not a literal type.